src/components/ground-nav/ground-nav.twig

```diff
@@ -10,101 +10,120 @@
+          {% with address %}
+            {{streetAddress}}<br>
+            {{addressLocality}}, {{addressRegion}} {{postalCode}} {{addressCountry}}<br>
+          {% endwith %}
+          <a href="mailto:{{email}}">{{email}}</a><br>
+          {# https://developers.google.com/web/fundamentals/native-hardware/click-to-call/ #}
+          <a href="tel:{{telephone|replace({
+            '.': '-',
+            '(': '-',
+            ')': '-',
+            ' ': ''
+          })}}">{{telephone}}</a>
+        </address>
       {% endwith %}
-      <a href="mailto:{{email}}">{{email}}</a><br>
-      {# https://developers.google.com/web/fundamentals/native-hardware/click-to-call/ #}
-      <a href="tel:{{telephone|replace({
-        '.': '-',
-        '(': '-',
-        ')': '-',
-        ' ': ''
-      })}}">{{telephone}}</a>
-    </address>
-  {% endwith %}
-  {# https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html #}
-  <ul role="list">
-    {% for item in social.items %}
-      <li>
-        <a href="{{item.link}}">
-          {# https://heydonworks.com/article/aria-label-is-a-xenophobe/ #}
-          <span class="u-hidden-visually">{{item.title}}</span>
-          {# http://simplyaccessible.com/article/7-solutions-svgs/#acc-heading-4 #}
-          {% include '@cloudfour/components/icon/icon.twig' with {
-            name: item.icon,
-            fallback: 'link',
-            focusable: 'false'
-          } only %}
-        </a>
-      </li>
-    {% endfor %}
-  </ul>
+      {# https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html #}
+      <div class="c-ground-nav__social">
+        <ul class="c-ground-nav__social-list o-list o-list--inline" role="list">
+          {% for item in social.items %}
+            <li class="class="c-ground-nav__social-item">
+              <a class="c-ground-nav__social-action" href="{{item.link}}">
+                {# https://heydonworks.com/article/aria-label-is-a-xenophobe/ #}
+                <span class="u-hidden-visually">{{item.title}}</span>
+                {# http://simplyaccessible.com/article/7-solutions-svgs/#acc-heading-4 #}
+                {% include '@cloudfour/components/icon/icon.twig' with {
+                  name: item.icon,
+                  fallback: 'link',
+                  focusable: 'false'
+                } only %}
+              </a>
+            </li>
+          {% endfor %}
+        </ul>
+      </div>
 
-  <nav aria-labelledby="footer-menu-label">
-    <h2 id="footer-menu-label" class="u-hidden-visually">
-      Extended Menu
-    </h2>
-    {# https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html #}
-    <ul role="list">
-      {% for item in menu.items %}
-        <li>
-          <a href="{{item.link}}"
-            {% if item.current %}
-              aria-current="page"
-            {% endif %}>
-            {{item.title}}
-          </a>
-        </li>
-      {% endfor %}
-    </ul>
-  </nav>
+      <hr class="c-ground-nav__divider u-hidden@m">
```

Comment:
I learned recently that horizontal rules have more semantic meaning and accessibility impact than I thought they did:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr
- https://www.w3.org/WAI/GL/WD-WCAG10-TECHS-20000615/Overview.html#structure

We should avoid using them and instead use CSS where possible.

I'd also encourage you to build the responsive hide/show functionality into the component itself where it's necessary to achieve the desired layout.